src/authentication/controller/authorization-controller.js

```diff
@@ -3,24 +3,34 @@ const passport = require('passport');
 require('./passport-config');
 
 exports.authorizeUser = (...roles) => (req, res, next) => {
-    passport.authenticate('jwt',{ session:false },
+    passport.authenticate('jwt',{ session: false },
        (error,payload) => {
            if(error) {
-              return res.status(400).json(error)
+            return res.status(400).json({
+              error: error,
+              data: [" "]
```

Comment:
When data is missing, make it null or empty array. We may come to this later and standardize further.
Apply to all occurances.